src/@chakra-ui/gatsby-plugin/theme.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  extendTheme,
+  type ThemeConfig,
+  type ThemeOverride,
+} from "@chakra-ui/react"
+
+// Global style overrides
+import styles from "./styles"
+
+// Foundational style overrides
+import foundations from "./foundations"
+
+const config: ThemeConfig = {
+  cssVarPrefix: "eth",
+  initialColorMode: "light",
+  useSystemColorMode: false,
```

Comment:
I think we should use the system color mode personally, I enjoy this feature when I use websites.

Reply:
Agreed. I've changed it 👍🏼